Docs/README.md

```diff
@@ -0,0 +1,127 @@
+Unmapped reads:                                0
+
+mapq < mapq_cut (non-unique):                  3127757
+mapq >= mapq_cut (unique):                     29616815
+Read-1:                                        14841738
+Read-2:                                        14775077
+Reads map to '+':                              14805391
+Reads map to '-':                              14811424
+Non-splice reads:                              25455360
+Splice reads:                                  4161455
+Reads mapped in proper pairs:                  21856264
+Proper-paired reads map to different chrom:    7648
+```
+###infer_experiment.py
+
+This scripts guesses how the reads were stranded for strand specific RNA-seq data. Example output:
+
+```
+infer_experiment.py -r hg19.refseq.bed12 -i Pairend_nonStrandSpecific_36mer_Human_hg19.bam
+
+#Output:
+
+This is PairEnd Data
+Fraction of reads failed to determine: 0.0172
+Fraction of reads explained by "1++,1--,2+-,2-+": 0.4903
+Fraction of reads explained by "1+-,1-+,2++,2--": 0.4925
+```
+
+###junction_saturation.py
+This scripts aims to determine if the used sequencing depth is enough to perform  alternative splicing analysis on the data. 
+
+![Junction Saturation](images/junction_saturation.png)
+
+In the above example output the red line (known junction) has reached a plateau. Nearly all known junctions in this data set have already been detected, and deeper sequencing will thus not help in detecting more known junctions. The green line (novel junction) has not plateaued and thus further sequencing will likely aid in junction saturation.
+In summary, if the lines are flattening out then you have sequenced deeply enough. 
+###RPKM saturation
+Resamples a subset of the total RNA reads and calculates the RPKM value for each subset. We use the default subsets of every 5% of the total reads. I.e 5%,10%...95%,100%. 
+A percent relative error is then calcuated based on the subsamples, this is the y-axis in the graph. 
+![RPKM Saturation](images/rpkm_saturation.png)
+
+
+###read duplication
+
+
+![Read duplication](images/read_duplication.png)
+
+###inner distance
+
+###gene body coverage
+
+###read distribution
+
+###junction annotation
+
+##dupRadar
+dupRadar is A Bioconductor library for R. 
+It plots the duplication rate per gene. A good sample (i.e. without PCR duplication?) will have a sigmodial shape as seen in the example output below:  
+
+##preseq 
+
+subread featureCounts - gene counts, biotype counts, rRNA estimation.
+##String Tie
+FPKMs for genes and transcripts
+## edgeR
+edgR is a bioconductor package for R and is used to do differntial extression analysis on RNAseq data.
+Rcreate MDS plot and sample pairwise distance heatmap / dendrogram
+[edgeR Bioconductor page](https://bioconductor.org/packages/release/bioc/html/edgeR.html)
+The output generated from edgeR process is a heatmap and a MDS-plot.
+The heatmap may look something like this:
+![Heatmap](images/heatmap.png =500x500))
```

Comment:
Not sure if this syntax is specific to Mou? Not seen size specifications like this before. Breaks it on GitHub though, and probably most markdown processors..